hugolib/site.go

```diff
@@ -99,7 +99,7 @@ type SiteInfo struct {
 	Social                SiteSocial
 	Sections              Taxonomy
 	Pages                 *Pages
-	Files                 []*source.File
+	Files                 *[]*source.File
```

Comment:
I made this change to match the `Pages *Pages` line above it - so I can get access to the `Site.Files` from somewhere that only only has the SiteInfo.

I did contemplate moving both Pages and Files from the Site struct to SiteInfo, but I'd rather not :)